Dfe.Academies.Academisation.Data/Repositories/ApplicationRepository.cs

```diff
@@ -78,5 +78,16 @@ private IQueryable<Application> DefaultIncludes()
 
 			return x;
 		}
+
+		public async Task SoftDelete(int applicationId)
+		{
+			var entity = await _context.Applications.FindAsync(applicationId);
+			if(entity != null){
+				 entity.DeletedAt = DateTime.Now;
```

Comment:
The use of DateTime.Now is correct, but in a distributed system, it's often better to use DateTime.UtcNow to avoid issues with different time zones.

Or what we do in conversions if use a DateTimeSource, then we're able to Mock it for testing.